### PR TITLE
Dustin/engine playground

### PR DESCRIPTION
**Pull request by DustinTurska**
## Problem solved

FIxes [DASH-495](https://linear.app/thirdweb/issue/DASH-495/engine-playground)

Short description of the bug fixed or feature added

Added Engine Section to playground
-Airdrop
-Mint
-Webhooks

Updated AppSidebar `expanded: false` to show `otherLinks` section - previously this was unseen by users and there was no scroll option

Create EngineAPIHeader for Engine CTA and utm link updates

Slack [thread](https://thirdwebdev.slack.com/archives/C07U1LM6S2W/p1731687004602369) for initiative

---

## PR-Codex overview
This PR introduces several enhancements and new features to the `playground-web` application, including new API routes for transaction handling, updates to the navigation structure, and the addition of several UI components for airdrop and minting functionalities.

### Detailed summary
- Added new environment variables to `turbo.json`.
- Updated `package.json` with new dependencies: `@thirdweb-dev/engine` and `timeago.js`.
- Modified `navLinks.ts` to collapse certain sections and added an "Engine" section.
- Implemented new pages for `webhooks`, `airdrop`, and `minting`.
- Created new API routes for transaction status and minting.
- Added components for `Webhook`, `Sponsorship`, `AirdropERC20`, and `ERC1155MintTo`.
- Enhanced UI with new card and table components for transaction results.

> The following files were skipped due to too many changes: `pnpm-lock.yaml`

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: apps/playground-web/src/app/api/airdrop/route.ts [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+import * as dotenv from "dotenv";
+import type { NextRequest } from "next/server";
+import { NextResponse } from "next/server";
+
+dotenv.config();
+
+const CHAIN_ID = "84532";
+const BACKEND_WALLET_ADDRESS = process.env.ENGINE_BACKEND_WALLET as string;
+
+console.log("Environment Variables:");
```
Comment (gregfromstl) [inline apps/playground-web/src/app/api/airdrop/route.ts:11]: Remove logs

File: apps/playground-web/src/app/api/airdrop/route.ts [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+        { error: "Invalid data format" },
+        { status: 400 },
+      );
+    }
+
+    if (data.length === 0) {
+      return NextResponse.json({ error: "Empty data array" }, { status: 400 });
+    }
+
+    console.log(`Attempting to mint batch to ${data.length} receivers`);
```
Comment (gregfromstl) [inline apps/playground-web/src/app/api/airdrop/route.ts:54]: More logs

File: apps/playground-web/src/app/navLinks.ts [hunk 3]
```diff
@@ -73,6 +73,32 @@ export const navLinks: SidebarLink[] = [
...
+      },
+      {
+        name: "Minting",
+        href: "/engine/minting",
+      },
+      {
+        name: "Webhooks",
+        href: "/engine/webhooks",
+      },
+      // {
```
Comment (gregfromstl) [inline apps/playground-web/src/app/navLinks.ts:92]: Remove these if we don't have them yet

File: apps/playground-web/src/components/engine/minting/erc1155-mint-to.tsx [hunk 4]
```diff
@@ -0,0 +1,301 @@
...
+      setIsUploading(false);
+    }
+  };
+
+  return (
+    <div className="flex flex-col items-center justify-center">
+      <div className="flex w-full flex-col gap-8 md:flex-row">
+        {/* Form Section */}
+        <div className="flex flex-col items-center justify-center">
+          <ConnectButton client={THIRDWEB_CLIENT} chain={baseSepolia} />
```
Comment (gregfromstl) [inline apps/playground-web/src/components/engine/minting/erc1155-mint-to.tsx:167]: Formatting on this button looks weird when no connected

File: apps/playground-web/src/components/engine/airdrop/TransactionResults.tsx [hunk 5]
```diff
@@ -0,0 +1,278 @@
...
+                    {(() => {
+                      if (!result.toAddress) return "----";
+
+                      const addressDisplay = `${result.toAddress.substring(0, 6)}...${result.toAddress.substring(
+                        result.toAddress.length - 4,
+                      )}`;
+                      // Keeping OP here for consistency. Will be adding a component for Optimism soon.
+                      const getExplorerUrl = () => {
+                        switch (result.network) {
+                          case "Base Sep":
```
Comment (gregfromstl) [inline apps/playground-web/src/components/engine/airdrop/TransactionResults.tsx:196]: Nit: probably enough room to spell out "Sepolia"

**Review by gregfromstl — APPROVED** [review]

File: apps/playground-web/src/app/api/airdrop/route.ts (hunk 1 above)
Comment (DustinTurska) [reply apps/playground-web/src/app/api/airdrop/route.ts]: Removed this file entirely, it is unused, removed logs from the rest as well

File: apps/playground-web/src/app/api/airdrop/route.ts (hunk 2 above)
Comment (DustinTurska) [reply apps/playground-web/src/app/api/airdrop/route.ts]: removed

File: apps/playground-web/src/app/navLinks.ts (hunk 3 above)
Comment (DustinTurska) [reply apps/playground-web/src/app/navLinks.ts]: done

File: apps/playground-web/src/components/engine/airdrop/TransactionResults.tsx (hunk 5 above)
Comment (DustinTurska) [reply apps/playground-web/src/components/engine/airdrop/TransactionResults.tsx]: Fixed, good call out

**Review by joaquim-verges — APPROVED** [review]
What's missing from these is the "how" of these engine demos. How do i build this for myself? was expecting to see some code snippets, or at least some high level endpoints that get called in those demos

@DustinTurska

**Pull request merged**